src/lib/bootloader/sysconfig.rb

```diff
@@ -61,6 +67,18 @@ def pre_write
         "# This setting is only relevant to UEFI which supports UEFI. It won't\n" \
         "# take effect on any other firmware type.\n" \
         "#\n" \
+        "#\n",
+
+      # FIXME: the default?
+      trusted_boot: "\n" \
+        "## Path:\tSystem/Bootloader\n" \
+        "## Description:\tBootloader configuration\n" \
+        "## Type:\tyesno\n" \
+        "## Default:\t\"no\"\n" \
+        "#\n" \
+        "# Enable Trusted Boot support\n" \
+        "# FIXME here we explain the benefits and distinction from Secure Boot\n" \
+        "# Only available for legacy (non-UEFI) boot.\n" \
```

Comment:
and only on certain architectures. Ideally I prefer to write it only where it make sense.